kerl

```diff
@@ -894,6 +899,17 @@ endif
 
 rehash
 ACTIVATE_CSH
+
+    otpver=$(get_otp_version "$rel")
+    if [ $otpver -ge 20 ]; then
+        # https://twitter.com/mononcqc/status/877544929496629248
+        cat << OTP_20_ADDITION >> "$absdir/activate"
+_KERL_SAVED_ERL_AFLAGS="\$ERL_AFLAGS"
```

Comment:
The issue you saw with deactivation not working properly comes from 2 parts:
 - if `$ERL_AFLAGS` is empty, `$_KERL_SAVED_ERL_AFLAGS` will be empty, the `[ -n "$_KERL_SAVED_ERL_AFLAGS" ]` check will fail, and `ERL_AFLAGS` will not be reset
 - we do not check that `$ERL_AFLAGS` does not already contain `-kernel shell_history` somewhere